collector/meminfo_numa_linux.go

```diff
@@ -127,8 +149,34 @@ func parseMemInfoNuma(r io.Reader) (map[meminfoKey]float64, error) {
 
 		// Active(anon) -> Active_anon
 		metric = re.ReplaceAllString(metric, "_${1}")
-		memInfo[meminfoKey{metric, parts[1]}] = fv
+		memInfo = append(memInfo, meminfoMetric{metric, prometheus.GaugeValue, parts[1], fv})
 	}
 
 	return memInfo, nil
```

Comment:
Can you add the scanner.Err() check here as well please. And we should add a len(parts) check above before parsing the value in order to not panic on an invalid meminfo file.